README.md

```diff
@@ -4,29 +4,78 @@
 [![Build Status](https://travis-ci.org/jupyter/nbgrader.svg)](https://travis-ci.org/jupyter/nbgrader)
 [![codecov.io](http://codecov.io/github/jupyter/nbgrader/coverage.svg?branch=master)](http://codecov.io/github/jupyter/nbgrader?branch=master)
 
-A system for assigning and grading notebooks.
+A system for assigning and grading Jupyter notebooks.
 
 [Documentation can be found on Read the Docs.](http://nbgrader.readthedocs.org)
 
+
+## Visual highlights of nbgrader
+
+### Instructor toolbar extension for Jupyter notebooks
+The nbgrader toolbar extension for Jupyter notebooks guides the instructor through
+assignment and grading tasks using the familiar Jupyter notebook interface.
+
+![nbgrader toolbar extension for Jupyter notebooks](docs/source/user_guide/images/assignment_toolbar.png "nbgrader toolbar extension for Jupyter notebooks")
+
+![Creating assignment](docs/source/user_guide/images/creating_assignment.gif "Creating assignment")
+
+### Student assignment list extension for Jupyter notebooks
+Using the assignment list extension, students may conveniently view, fetch,
+submit, and validate their assignments.
+
+![nbgrader assignment list](docs/source/user_guide/images/student_assignment.gif "nbgrader assignment list")
+
+### Using formgrader
```

Comment:
Let's actually leave this section out for now rather than have a placeholder image -- I'll work on creating some better graphics of the formgrader and then we can make more changes to this PR later.